users/vieting/experiments/switchboard/hybrid/feat/network_helpers/perturbation.py

```diff
@@ -0,0 +1,80 @@
+class PerturbationFactor:
+  """
+  Class to wrap perturbation factors, e.g. for speed or tempo perturbation.
+  """
+  def __init__(self, prob, minimum, maximum):
+    self.prob = prob
+    self.min = minimum
+    self.max = maximum
+
+
+class WaveformPerturbation:
+  """
+  Helper class to perform perturbation techniques on audio waveforms.
+  """
+  def __init__(self, speed=None, tempo=None, sox_effects=None, codecs=None, preemphasis=None):
+    """
+    :param speed:
+    """
```

Comment:
This docstring is incomplete. It would be nice to add some short description and examples for each argument.

Reply:
Done